doc/design.md

```diff
@@ -101,7 +101,7 @@ spec:
 We hope Fluid users could represent it by the following line.
 
 ```python
-skaffold_git = fluid.Git(
+skaffold_git = fluid.git_resource(
```

Comment:
Why not use `fluid.git` for simple?

Reply:
> Why not use fluid.git for simple?

Just tracing the current implementation. I think we can update the API and design in another PR if need.

Reply:
I remember that Tekton has a limited number of pre-defined resource types and git is one of them. I would suggest we keep it Git other than git_resource, because git_resouce is not a fullname; git_pipeline_resource is. But git_pipeline_resource is too long. It seems reasonable to use a short name Git for one of a few pre-defined types.